### ISSUE TITLE
[BUG] Performance API Endpoint does not return grossPerformance anymore

### ISSUE THREAD

**Issue by brhahlen** [ghostfolio/ghostfolio#3750]
**Bug Description**

I use [Homepage ](https://gethomepage.dev/latest/) which has a widget for [Ghostfolio](https://gethomepage.dev/latest/widgets/services/ghostfolio/).
The code for the widget is [here](https://github.com/gethomepage/homepage/blob/main/src/widgets/ghostfolio/component.jsx).
This widget uses the Performance API endpoint. Since a recent release of Ghostfolio, this does not return the `grossPerformancePercentage` value anymore. This causes Homepage to return a NaN% value.

When I do a manual curl of the endpoint, I do not get the data either:
 ```
$ curl https://ghostfolio/api/v2/portfolio/performance?range=1y -H "Authorization: Bearer TOKEN" | jq -r .performance

{
  "netPerformance": 225.03713280104142,
  "netPerformanceWithCurrencyEffect": 219.05359220814364,
  "totalInvestment": 2439.044084997045,
  "currentNetWorth": 0,
  "currentValueInBaseCurrency": 2497.538049299246,
  "netPerformancePercentage": 0.11679226620450237,
  "netPerformancePercentageWithCurrencyEffect": 0.1136868619671092
}
```

Was this an intentional change?

**To Reproduce**

1. curl https://ghostfolio/api/v2/portfolio/performance?range=1y -H "Authorization: Bearer TOKEN" | jq -r .performance

**Expected behavior**

It should return the grossPerformance

**Environment**

- Ghostfolio Version 1.107.0
- Self-hosted Docker
- Experimental Features enabled
- Firefox

**Comment by brhahlen** [comment]
Hi @dtslvr ,

Thanks for the response. The Public API only seems to be for importing data, not getting any other data out.
Are there plans to supply a Public API for this purpose?

Thanks,
Ben

**Comment by dtslvr** [comment]
> Are there plans to supply a Public API for this purpose?

Yes, here is a proposal for a new _Public API_ endpoint: https://github.com/ghostfolio/ghostfolio/pull/3762

**Comment by brhahlen** [comment]
> > Are there plans to supply a Public API for this purpose?
> 
> Yes, here is a proposal for a new _Public API_ endpoint: #3762

Cool, I'll have a look at that!
Thanks!
